DTS-STN/canadian-dental-care-plan
# Removing dental insurance lookup service as well as corresponding mocks and configuration

### Description
Follow-up PR for https://github.com/DTS-STN/canadian-dental-care-plan/pull/882, there is now no need for a lookup service function to get the dental insurance options.

Cache time configuration and Lookup API/DB mocks have also been removed.

### Checklist

### Test Instructions
Run the application and start a CDCP application. Continue the application until you reach `/dental-insurance`. There should be no issues with the "Yes"/"No" options when submitting this page and the result of your selection should show on the `/review-information` page.
